packages/assistify-help-request/client/views/tabbar/HelpRequestActions.js

```diff
@@ -44,7 +50,7 @@ Template.HelpRequestActions.events({
 		swal(_.extend({
 			title: t('Closing_chat'),
 			text: warnText ? t(warnText) : '',
-			type: 'input',
+			type: showClosingComment(),
```

Comment:
This is kind-of-a-hack. The swal-configuration could also not contain the input properties. Let me give you a sample

```
let swalConfig = {
			title: t('Closing_chat'),
			type: showClosingComment(),
			showCancelButton: true,
			closeOnConfirm: false
		};

		if (showClosingComment()) {
			swalConfig = _.extend(swalConfig, {
				type: 'input',
				inputPlaceholder: t('Please_add_a_comment')
			});
		}

		swal(swalConfig, (inputValue) => {
```